Jenkinsfile

```
@@ -0,0 +1,83 @@
+pipeline {
+    agent any
+
+    stages {
+
+        stage('Setting version') {
+            steps {
+                sh "./gradlew makeVersionFile"
+                script {
+                    def version = readFile('build/version')
+                    currentBuild.displayName = "${version} (#${env.BUILD_NUMBER})"
+                }
+            }
+        }
+
+        stage("Clean") {
+            when {
+                anyOf {
+                    branch "master*"
+                    changeRequest target: 'master'
```

Can we target the same as the `branch` instruction?
```suggestion
                    changeRequest target: 'master*'
```

---

:+1: